.changeset/afraid-shoes-cough.md

```diff
@@ -0,0 +1,5 @@
+---
+"@justeattakeaway/pie-link": minor
+---
+
+Updated visual tests to split by size
```

Comment:
Probably wouldn't create a changeset for this as it result in a package bump that doesnt affect consumers.